homework-custom-lodash/helpers/combineObj.js

```diff
@@ -0,0 +1,6 @@
+const combineObj = function (obj1,obj2) {
+    const resultObj = {...obj1,...obj2};
```

Comment:
spaces, 

please configure ESlint with airbnb styleguide to avoid such issues in the future

Reply:
okay

Reply:
i install eslint and configure it